site/en/guides/border-router/mdns-discovery.md

```diff
@@ -0,0 +1,123 @@
+# mDNS Discovery
+
+OTBR registers a `_meshcop._udp` service named `OpenThread_BorderRouter`. This
+service is configured in the [OTBR CMakeLists.txt file](https://github.com/openthread/ot-br-posix/blob/main/CMakeLists.txt#L36).
+
+There are several ways to discover your Thread network.
+
+## DNS Service Discovery (DNS-SD)
+
+Use `dns-sd` to browse for a service on `raspberrypi.local.:49155`:
+
+```
+$ dns-sd -L "OpenThread_BorderRouter" _meshcop._udp local
+```
+
+<pre>
+Lookup OpenThread_BorderRouter._meshcop._udp.local
+DATE: ---Tue 16 Nov 2021---
+13:31:03.197  ...STARTING...
+13:31:03.350  OpenThread_BorderRouter._meshcop._udp.local. can be reached at raspberrypi.local.:49155 (interface 3)
+</pre>
+
+Get the IP addresses associated with your OTBR domain name:
+
+```
+$ dns-sd -G v4/v6 {raspberrypi.local}
+```
+
+<pre>
+DATE: ---Tue 16 Nov 2021---
+14:21:29.485  ...STARTING...
+Timestamp     A/R Flags if Hostname           Address                                      TTL
+14:21:29.486  Add     3  3 raspberrypi.local. FDDE:AD11:11DE:0000:74D0:6FC9:6BE6:3582%&lt;0&gt;  120
+14:21:29.486  Add     3  3 raspberrypi.local. FD00:0000:0000:0000:74D0:6FC9:6BE6:3582%&lt;0&gt;  120
+14:21:29.486  Add     3  3 raspberrypi.local. FE80:0000:0000:0000:74D0:6FC9:6BE6:3582%eth0 120
+14:21:29.486  Add     3  3 raspberrypi.local. FE80:0000:0000:0000:287F:87CA:F4B3:498A%eth0 120
+14:21:29.486  Add     2  3 raspberrypi.local. 192.168.0.10                                 120
+</pre>
+
+Ping your IP address. From the `dns-sd` results, choose an address that's
+reachable from your network, for example the global `<0>` address
+**FD00:0000:0000:0000:74D0:6FC9:6BE6:3582**:
+
+```
+$ ping -6 FD00:0000:0000:0000:74D0:6FC9:6BE6:3582
+
+```
+
+<pre>
+PING FD00:0000:0000:0000:74D0:6FC9:6BE6:3582(fd00::74d0:6fc9:6be6:3582) 56 data bytes
+64 bytes from fd00::74d0:6fc9:6be6:3582: icmp_seq=1 ttl=64 time=27.1 ms
+64 bytes from fd00::74d0:6fc9:6be6:3582: icmp_seq=2 ttl=64 time=3.18 ms
+64 bytes from fd00::74d0:6fc9:6be6:3582: icmp_seq=3 ttl=64 time=2.76 ms
+</pre>
+
+## Avahi utilities
+
+Install `avahi-daemon` and `avahi-utils`:
+
+```
+$ sudo apt-get install -y avahi-daemon avahi-utils
+```
+
+Start the `avahi-daemon`:
+
+```
+$ sudo service avahi-daemon start
+```
+
+Use `avahi-browse`. To resolve IPv6 only, edit `/etc/avahi/avahi-daemon.conf`
+and set `use-ipv4=no` and `use-ipv6=yes`.
```

Comment:
It's not necessary to limit the usage to IPv6 only.
The Border Agent supports both IPv4 and IPv6 connections. If you start the commissioner with the IPv4 address in the example below (i.e. `192.168.0.10`) it should just work.

Reply:
Ah, OK - I was wondering about that. Thanks, I made some updates.